packages/terra-i18n/src/intlLoaders.js

```diff
@@ -1,29 +1,53 @@
-/* eslint-disable */
+/* eslint-disable import/no-unresolved, compat/compat, no-console */
 import intlLoaders from 'intlLoaders';
 
+const hasIntlData = (locale) => {
+  const intlConstructors = [
+    Intl.DateTimeFormat,
+    Intl.NumberFormat,
+  ].filter((intlConstructor) => intlConstructor);
+
+  if (intlConstructors.length === 0) {
+    return false;
+  }
+
+  return intlConstructors.every((intlConstructor) => {
+    const supportedLocalesFromProvidedList = intlConstructor.supportedLocalesOf([locale], { localeMatcher: 'lookup' });
+    return !!supportedLocalesFromProvidedList.length && supportedLocalesFromProvidedList[0] === locale;
```

Comment:
I didn't know you can use `!!` to convert it.

Reply:
I recently discovered it in another project!